docs/mkdocs.yml

```diff
@@ -52,9 +57,9 @@ theme:
   font:
     code: Fira Code
     text: Montserrat
-  language: en
+  language: custom
```

Comment:
This was required for the search to set some properties, see [this](https://github.com/fragcolor-xyz/shards/blob/621844d6662c12842b30a5c004054651f9b1594a/docs/overrides/partials/languages/custom.html) file

The main thing of interest is `"search.config.pipeline": "stopWordFilter",` since it removes the other default filter that filters out symbols, so you couldn't search for ">=", ">>",etc. for example

Unless there's an other way to set this (couldn't figure it out), I'm setting it this way. If we ever decide to build other language documentation we can split this.